src/screens/TransactionDetailsScreen.tsx

```diff
@@ -202,6 +229,20 @@ export class TransactionDetailsScreen extends Component<Props, State> {
   }
 }
 
+const mapStateToProps = (state: ApplicationState) => ({
+  transactions: Object.values(state.transactions.transactions),
```

Comment:
If you have transactions as `Record<string, Transaction>` and the key is a transaction hash, then you can easily access them by
```
transactions[hash]
```

you will not need
```
props.transactions.filter(transaction => {
      if (transaction.hash == hash) {
```